evap/evaluation/templates/legal_notice.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Legal Notice" %} - {{ block.super }}{% endblock %}
+
+{% block subtitle %}
+    {{ block.super }}
+    <li><a href="{% url "evap.evaluation.views.legal_notice" %}">{% trans "Legal Notice" %}</a></li>
```

Comment:
is this the breadcrumb? if yes, please remove the link. i think there shouldn't be any other pages which have a link on themselves.